lib/src/matomo.dart

```diff
@@ -429,31 +414,17 @@ class MatomoTracker {
 
   /// Tracks an event.
   ///
-  /// [eventCategory] corresponds with `e_c`, [action] with `e_a`, [eventName] with `e_n`
-  /// and [eventValue] with `e_v`.
-  ///
-  /// Here are some typical examples for what each parameter usually describes:
-  /// - `eventCategory`: Videos, Music, Games...
-  /// - `action`: Play, Pause, Duration, Add Playlist, Downloaded, Clicked...
-  /// - `eventName`: Movie name, or Song name, or File name...
-  ///
   /// For remarks on [dimensions] see [trackDimensions].
   void trackEvent({
-    required String eventCategory,
-    required String action,
-    String? eventName,
-    int? eventValue,
+    required EventInfo eventInfo,
     Map<String, String>? dimensions,
   }) {
     validateDimension(dimensions);
     return _track(
       MatomoEvent(
         tracker: this,
-        action: action,
-        eventAction: action,
-        eventName: eventName,
-        eventCategory: eventCategory,
-        eventValue: eventValue,
+        action: eventInfo.action,
```

Comment:
A `MatomoEvent` representing an event should not have an `action` set, otherwise, the `MatomoEvent` would trigger two *Matomo actions*: an event and a page view. (*Matomo actions* w.r.t the linked FAQ).